src/chem/Complex.js

```diff
@@ -1084,72 +1097,62 @@ class Complex {
     this._bonds = [];
     this._sgroups = [];
 
+    const self = this;
     let atomBias = 0;
     let bondBias = 0;
     let residueBias = 0;
     let chainBias = 0;
     let componentBias = 0;
-    for (i = 0; i < complexes.length; ++i) {
-      const c = complexes[i];
 
-      // add atoms
-      for (j = 0; j < c._atoms.length; ++j) {
-        const a = c._atoms[j];
-        a._serial += atomBias;
-        a._index += atomBias;
-        this._atoms.push(a);
-      }
+    function processAtom(atom, _atomBias) {
```

Comment:
We start param name with '_' only for unused paramaters. We should use here simple 'bias'.